CONTRIBUTING.md

```diff
@@ -52,6 +52,7 @@ infrastructure or services should run in this project.
 
     ```text
     gsutil mb -p ${PROJECT_ID} gs://${BUCKET_ID}
+    gsutil versioning set on gs://${BUCKET_ID}
```

Comment:
I'm not sure we want to encourage versioning. Maybe move it to it's own "```text" box with more context. Versioning can increase costs.

Reply:
It is enabled by default when you run using bootstrap - these instructions just make it so you create it correctly for tests.